src/mono/dlls/mscordbi/CMakeLists.txt

```diff
@@ -124,6 +124,7 @@ include_directories(${CLR_DIR}/minipal)
 
 add_subdirectory(${CLR_DIR}/md/enc md/enc)
 add_subdirectory(${CLR_DIR}/utilcode utilcode)
+#add_subdirectory(${CLR_DIR}/debug/dbgutil /debug/dbgutil)
```

Comment:
if this dependency is not required please delete the line instead of commenting it out